package.json

```diff
@@ -17,7 +17,7 @@
     "bump": "node ./scripts/binary.js bump",
     "check-node-version": "node scripts/check-node-version.js",
     "check-terminal": "node scripts/check-terminal.js",
-    "clean": "lerna run clean --parallel",
+    "clean": "lerna run clean --parallel --no-bail",
```

Comment:
(can't comment on the right line, but:) I think the "test" command also needs to be updated so the `config` package tests run in CI:

```
"test": "yarn lerna exec yarn test --scope cypress --scope \"'@packages/{config,electron,extension,https-proxy,launcher,net-stubbing,network,proxy,rewriter,runner,runner-shared,socket}'\"",
```

Right now I'm not sure they run in CI. I don't see them in `unit-tests`: https://app.circleci.com/pipelines/github/cypress-io/cypress/26077/workflows/af61706e-6d22-4391-bcf9-4c07335b7ac6/jobs/981791

(why do we have a `test-unit` and a `test` command? no good reason)

Reply:
good call: [`b351eda` (#18589)](https://github.com/cypress-io/cypress/pull/18589/commits/b351edaf33a80d65dcf08ca893e250193179cc4a#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R49)